app/views/steps/outgoings/outgoings_more_than_income/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= t('steps.outgoings.caption') %></span>
+     <%= step_form @form_object do |f| %>
+        <%= f.govuk_radio_buttons_fieldset :outgoings_more_than_income, legend: { text: t('.outgoings_more_than_income'), tag: 'h1', size: 'xl' } do %>
+          <% @form_object.choices.each_with_index do |choice, index| %>
+            <% if choice == YesNoAnswer::YES %>
```

Comment:
Don't think we need to the literal comparison like this as you can ask a "choice" what it is directly:

e.g.

`choice.yes?`
>>> true

Reply:
Roger that. I knew this was how it's set up, hadn't worked in a previous view (which it is now) I think bc I hadn't hooked up the types correctly and had copied the same logic over. All working now, all good - thanks for pointing out